### ISSUE TITLE
limitless folders

### ISSUE THREAD

**Issue by DI555** [Telegram-FOSS-Team/Telegram-FOSS#472]
i just found that not possible to create more than 10 folders(((.
This should be hardly fixed!!!

**Comment by Younes-L** [comment]
Agree, however it's a server limitation by telegram and on top of that tg-foss isn't meant to add features.
There are other fork clients that have limitless, local only folders (since server limit is always 10 and cannot be changed by client).
You can read about all sorts of telegram limits here
[limits.tginfo.me](https://limits.tginfo.me) (Ru)
[limits.tginfo.me/en](https://limits.tginfo.me/en) (En)

**Comment by DI555** [comment]
nope, i see folders are not in that list, so this issue may be fixed by the code !!!
i see the simillar my issue that could be fixed this way either!!! -
https://github.com/Telegram-FOSS-Team/Telegram-FOSS/issues/341

**Comment by rustforfuture** [comment]
The bad things about this is those additional Folders won't sync woth telegram servers.

And it may cause issues in future.

Like what plus messenger did to me.

It has a local folder option and when i deleted those, i saw that some of ky cloud folders that i paintakenly sorted and gathered them renoved completely.

It was terrible.
